lib/awful/widget/watch.lua

```diff
@@ -0,0 +1,84 @@
+---------------------------------------------------------------------------
+--- Watch widget.
+--
+-- @release @AWESOME_VERSION@
+-- @classmod awful.widget.watch
+---------------------------------------------------------------------------
+
+local setmetatable = setmetatable
+local textbox = require("wibox.widget.textbox")
+local timer = require("gears.timer")
+local spawn = require("awful.spawn")
+
+local watch = { mt = {} }
+
+local function easy_async_spawn(cmd, callback)
+  local stdout = ''
+  local stderr = ''
```

Comment:
i will update this PR after/if this one will be merged: https://github.com/awesomeWM/awesome/pull/668